tests/func/test_data_cloud.py

```diff
@@ -247,41 +240,23 @@ def _setup_cloud(self):
 
         self.assertIsInstance(self.cloud.get_remote(), self._get_cloud_class())
 
-    def _get_url(self):
-        return GCP.get_url()
-
     def _get_cloud_class(self):
         return RemoteGS
 
 
-class TestRemoteAZURE(TestDataCloudBase):
-    def _should_test(self):
-        return Azure.should_test()
-
-    def _get_url(self):
-        return Azure.get_url()
-
+class TestRemoteAZURE(Azure, TestDataCloudBase):
     def _get_cloud_class(self):
         return RemoteAZURE
 
 
-class TestRemoteOSS(TestDataCloudBase):
-    def _should_test(self):
-        return OSS.should_test()
-
-    def _get_url(self):
-        return OSS.get_url()
-
+class TestRemoteOSS(OSS, TestDataCloudBase):
     def _get_cloud_class(self):
         return RemoteOSS
 
 
-class TestRemoteLOCAL(TestDataCloudBase):
-    def _should_test(self):
-        return Local.should_test()
-
+class TestRemoteLOCAL(Local, TestDataCloudBase):
     def _get_url(self):
-        self.dname = Local.get_url()
+        self.dname = super()._get_url()
         return self.dname
```

Comment:
Oh, this one doesn't match directly with LOCAL.get_url 🙁 I guess the line that uses `dname` down below is a pretty useless test, but just not to dive into it, we could simply wrap `super().get_url` here.